module-2/backend/week_three.md

```diff
@@ -10,14 +10,45 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 1. What is the entry at the command line to create a new rails app?
+rails new project_name_here"
+
 2. What do Models generally inherit from in rails?
+active record
+
 3. What do Controllers generally inherit from in a rails project?
+application controller
+
 4. How would I create a route if I wanted to see a specific horse in my routes fitle assuming I'm sticking to standard conventions and that I didn't want other CRUD functionality?
+get 'horses' => 'horse#show' OR resources :horses, only: [:show]
+
 5. What rake task is useful when looking at routes, and what information does it give you?
+rake routes tells you all the wonderful routes you have available to you
+
 6. What is an example of a route helper? When would you use them?
+nested routes. When things can't exist without other things it would be useful to have nested routes
```

Comment:
Route helpers are things like `horses_path` or `horses_url`. We generally use them in tests and when creating links.